adapter/noop/config/BUILD

```diff
@@ -0,0 +1,24 @@
+load("@org_pubref_rules_protobuf//gogo:rules.bzl", "gogoslick_proto_library")
```

Comment:
do we even need config for this?  can we just use google.protobuf.Empty and save this bit of BUILD ?